src/user_account/models.py

```diff
@@ -15,3 +15,18 @@ class User(AbstractUser):
     def update_score(self):
         self.avr_score = self.test_results.aggregate(points=Sum('avr_score')).get('points', 0.0) / \
                          self.test_results.count()
+
+    def count_passed_tests(self):
+        return self.test_results.filter(is_completed=True).count()
+
+    def total_score(self):
+        return self.avr_score
+
+    def last_run(self):
+        return self.test_results.last().datetime_run
+
+    def percent_success_passed(self):
+        percent_success_passed = sum([user.percent_correct_answers() for user in
+                                      self.test_results.filter(is_completed=True)])
```

Comment:
With such calculation you can easily end up with > 100%

Reply:
Fixed.